src/Parser/Monk/Mistweaver/CHANGELOG.js

```diff
@@ -1,4 +1,5 @@
 export default `
+17-10-2017 - File structure updates for module. Should be no impact to end user (by anomoly)
```

Comment:
I thought the CHANGELOG files were supposed to be specifically for the benefit of the end user. This shouldn't have a changelog entry.

Reply:
Sounds good. I know I had added something to the MW last time we did a refactor, but I will remove.